ftml/test/anchor2-alias-wrap.json

```diff
@@ -0,0 +1,48 @@
+{
+    "input": "[[anchor_]]Internal //elements//[[/anchor]]",
+    "tree": {
+        "elements": [
+            {
+                "element": "container",
+                "data": {
+                    "type": "paragraph",
+                    "elements": [
+                        {
+                            "element": "anchor",
+                            "data": {
+                                "attributes": {},
+                                "target": "same",
```

Comment:
(Out of scope for this PR) Does FTML distinguish between "same as specified by the syntax in this link" and "same by default"? If not, how would the API consumer know which links should override e.g. a [`<base target="_blank">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base) on the webpage?

Reply:
I was not aware that `<base>` was a tag before you mentioned this. I'm not really sure what you mean by

> "same as specified by the syntax in this link" and "same by default"

So I'm not exactly certain what the problem being identified here is.

Reply:
On a page with `<base target="_blank">`, I would expect all links to, by default, open in a new tab. (This could happen with a browser extension that makes the default link target _blank, possibly)

If I take my Wikitext for that page's content, send it to FTML, and FTML sends back the AST for me to render, and a given link in the tree has `target: "same"`, I don't know whether or not to ignore that (because it's the default, and all links on this page should open in a new tab), or obey that (because this link clearly wants to override the base behaviour).

Reply:
Interesting, this changes my understanding of links.

Now that there's a very clear case for `Option<AnchorTarget>` as the field type, I will be adjusting how links handle this and returning `None` (JSON `null`) when it's not explicitly set.